src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  background-color: #fff;
+
+  &__image {
```

Comment:
The BEM naming convention suggests using a single underscore for element names, like `&-image` instead of `&__image`. Consider changing this to maintain consistency with BEM.